plugins/modules/vmware_dvswitch.py

```diff
@@ -905,6 +918,13 @@ def update_dvswitch(self):
             results['net_flow_idle_flow_timeout'] = self.netFlow_idle_flow_timeout
             results['net_flow_sampling_rate'] = self.netFlow_sampling_rate
             results['net_flow_internal_flows_only'] = self.netFlow_internal_flows_only
+
+            results['net_flow_switch_ip'] = self.netFlow_switch_ip
+            if self.dvs.config.switchIpAddress != self.netFlow_switch_ip:
+                changed = changed_settings = True
+                changed_list.append("net_flow_switch_ip")
+                results['net_flow_switch_ip_previous'] = self.dvs.config.switchIpAddress
+                config_spec.switchIpAddress = self.netFlow_switch_ip
```

Comment:
Was first trying to do that in the beginning, but I couldn't figure out so far how I could split it out of the result then, because it's not in `vim.dvs.VmwareDistributedVirtualSwitch.IpfixConfig()` but one layer above in `vim.dvs.VmwareDistributedVirtualSwitch.ConfigSpec()`.
Will put some more time into it to find out how to put that into there 👍🏻

Reply:
@mariolenz
Sorry for the long delay, life happened 🙂 
I think I managed to merge it into the netflow checks function!
Manual testing is ongoing right now, but it should be all good.